### PR TITLE
remove pattern matching rule from `bandwagoner`

### LINKED ISSUE #75: code review of Bandwagoner suggests two conflicting
In the [Bandwagoner exercise](https://exercism.org/tracks/elm/exercises/bandwagoner/edit), considering the function type annotation requires fields `stats.wins` and `stats.losses` to exist, I wrote the following code:

```elm
rootForTeam: {a|stats: {wins: Int, losses: Int}} -> Bool
rootForTeam team =
    let {wins, losses} = team.stats
    in wins > losses
```

This code passes the tests, however, at code review, I get the following message:

> *Essential*
>
> Like the instructions mention, you should use pattern matching to get stats from the function parameter in rootForTeam.

So, new trial with a pattern match, i.e. with the following code:

```elm
rootForTeam: {a|stats: {wins: Int, losses: Int}} -> Bool
rootForTeam team =
    case team.stats of
        {wins, losses} -> wins > losses
```

then I get the following message:

> *Essential*
>
> Like the instructions mention, you should use pattern matching to get stats from the function parameter in rootForTeam.
>
> *Recommended*
>
> A way to simplify your code has been found.
>
> Simplify: Use a let expression to destructure data
> 
> ```elm
> 27|     case team.stats of
> 28|         {wins, losses} -> wins > losses
>             ^^^^^^^^^^^^^^
> ```
>
> It is more idiomatic in Elm to use a let expression to define a new variable rather than to use pattern matching. This will also make the code less indented, therefore easier to read.

---

I understand the recommandation, as using `let` was in fact my first bet.

But what about the essential message, as I now use the pattern match? Isn't this a false error?

Thanks for your feedback, and for Exercism!

### PR DESCRIPTION
**Pull request by jiegillet**
Closes #75 
[Sister PR here](https://github.com/exercism/website-copy/pull/2298).

As discussed in the issue, this is too early in the concept tree to require a pattern match/destructure.

### CODE REVIEW

**Review by ceddlyburge — APPROVED** [review]

**Comment by jiegillet** [comment]
@exercism/maintainers-admin Can I get an approval please? 💛

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
